.github/workflows/pr.yml

```diff
@@ -1,12 +1,12 @@
 name: PR
 on: [pull_request]
 jobs:
-  type-test:
-    name: Type test
+  type-check:
+    name: type-check
```

Comment:
🐧 this is what appears in the UI which is why I prefer something more human readable

Reply:
right - that is why I changed them to align them with the `npm run` target names. If a test fails, there is one less layer or translation from the web UI to the CLI command to test locally.